src/main/java/org/manifold/compiler/front/TupleTypeValueVertex.java

```diff
@@ -0,0 +1,114 @@
+package org.manifold.compiler.front;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.manifold.compiler.TypeTypeValue;
+import org.manifold.compiler.TypeValue;
+import org.manifold.compiler.Value;
+
+import com.google.common.collect.ImmutableMap;
+
+public class TupleTypeValueVertex extends ExpressionVertex {
+
+  private static Logger log = LogManager.getLogger("TupleTypeValueVertex");
```

Comment:
Similar to above, lets use the class so any static refactor can also change this.